doc/nrf/gs_updating.rst

```diff
@@ -29,14 +29,38 @@ If you disabled automatic updates:
 #. The :guilabel:`Update` button appears when an update is available.
    Click on the button to install the update.
 
-The |VSC| extension lets you update west and the associated |NCS| repositories within its :guilabel:`Source Control` panel.
-For detailed instructions, see the `West integration`_ section of the extension's documentation.
-
 .. _west_update:
 
 Updating west
 *************
 
+|method_note|
+
+Regardless of the method, west is always updated to the latest available version in the PyPi repository.
```

Comment:
Hey woah, this isn't quite right. VSC provides no mechanism for upgrading the west tool itself. The section in the link at the bottom is correct though. VSC lets you switch your NCS version and its dependencies through the source control panel, but the tool itself must be updated through other means

Reply:
Are the "other means" of updating west basically just `pip3 install -U west` and equivalents for each platform?

Reply:
I think that section relates more to updating everything that west manages, not west itself. I guess I misunderstood that at first.

Reply:
@trond-snekvik , pushed an edit, please check again.